src/main/kotlin/org/jitsi/jibri/sipgateway/pjsua/PjsuaClient.kt

```diff
@@ -40,6 +43,9 @@ class PjsuaClient(
 ) : SipClient() {
     private val logger = createChildLogger(parentLogger)
     private val pjsua: JibriSubprocess = JibriSubprocess(logger, "pjsua")
+    private val sipOutboundPrefix: String? by optionalconfig(
```

Comment:
Instead of making this optional and putting the default in the code, I'd make it non-optional and put the default value of `sip` in the default config file.

Reply:
Oops, sorry I misread this...I see this isn't the scheme.